nixos/modules/services/misc/gerrit.nix

```diff
@@ -0,0 +1,158 @@
+{ config, lib, pkgs, ... }:
+
+with lib;
+
+let
+
+  cfg = config.services.gerrit;
+
+  cmd = "${pkgs.jre_headless}/bin/java -jar ${cfg.package}/webapps/gerrit-${cfg.package.version}.war";
+
+  mkPluginSectionName = plugin: ''plugin "${escape [ "\"" ] plugin}"'';
+
+  settingsFile = pkgs.writeText "gerrit.config" ''
+    ${generators.toINI {} cfg.settings}
+    ${generators.toINI { mkSectionName = mkPluginSectionName; } cfg.pluginSettings}
+  '';
```

Comment:
I propose to import the git config generator from home-manager into the generators. It's quite close to the INI format but not entirely.

https://github.com/rycee/home-manager/blob/master/modules/programs/git.nix

Reply:
That would be neat, but this might be a conversation for #75584.

Reply:
That would be neat, but this might be a conversation for #75584.